bin/permadelete_crash_data.py

```diff
@@ -77,10 +78,15 @@ def cmd_permadelete(ctx, crashidsfile):
 
     for crashid in crashids:
         click.echo(f"Working on {crashid!r}")
+        start_time = datetime.datetime.now()
         # delete from storage
+        click.echo(">>> Deleting from storage ...")
         crashstorage.delete_crash(crashid)
         # delete from es
+        click.echo(">>> Deleting from Elasticsearch ...")
         es_crashstorage.delete_crash(crashid)
+        end_time = datetime.datetime.now()
+        click.echo(f">>> Time: {end_time - start_time}")
```

Comment:
I was on the fence about whether to adjust `delete_crash` so it indicated whether it deleted anything or not. If it didn't delete anything, then that would indicate to us that maybe we got the crash id wrong.

I was also on the fence about whether to hit the `CrashVerify` endpoint and have it spit out the status of the crash report's data. That seems like a good check to do.

@relud What do you think?

Reply:
those both seem like good things to have in a crash deletion report